packages/core/base/core-base-server-internal/src/regexp.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const uuidRegexp =
```

Comment:
Yea... To be honest from my experience, such code splitting always need some kind of 'misc' / 'put-stuff-here' low-level packages to work-around some bad isolation concerns that would otherwise be addressed by creating a dedicated package for each single constant. If I definitely don't like being forced to move that regexp here, I think it's somehow inevitable that a few 'shared' part of our code will be moved into these 'base'/'misc' packages. 

We can still revisit these `base` package once everything has been moved into packages. we would likely be able to improve things, but I do think we need to wait until the end of this migration to try to improve our packages tree, given we just don't have the full picture right now ihmo.